tests/Dockerfile

```diff
@@ -55,6 +55,7 @@ RUN printf '[user]\n\t\nemail = cockpituous@cockpit-project.org\n\tname = Cockpi
     ln -s /cache/images /build/images && \
     ln -s /cache/images /build/virt-builder && \
     ln -s /cache/github /build/github && \
+    ln -s /tmp /build/tmp && \
```

Comment:
OOI, is /build/tmp/ still being used anywhere, or is this just a safety net in case we forgot a spot?